templates/zerver/help/make-an-announcement.md

```diff
@@ -15,15 +15,15 @@ Announcements on Zulip are sent using the `#announce` stream.
 4. Enter the topic for your announcement in the **Topic** field.
 
     !!! tip ""
-        Your topic name can be no longer than 52 characters.
+        Your topic name cannot be longer than 52 characters.
 
-5. Enter your announcement in the box labeled
- **Compose your message here...** or messaging box.
+5. Enter your announcement in **compose box ** (also known as the messaging box).
```

Comment:
I think `**Compose your message here...**` was appropriate here with `(also known as the compose box)`. I wouldn't use `messaging box`, since it's confusing to use more than one name for the same element.